queries.sql

```diff
@@ -8,3 +8,28 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT *FROM animals WHERE neutered = 'True';
 SELECT *FROM animals WHERE name != 'Gabumon';
 SELECT *FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon%';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth  > '2022-01-02';
+SAVEPOINT delete_2022;
+UPDATE animals SET  weight_kg = ( weight_kg * -1);
+ROLLBACK TO delete_2022;
+UPDATE animals SET  weight_kg = ( weight_kg * -1) WHERE id IN (7,5,8,6);
+COMMIT;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Update%20all%20animals%27%20weights%20that%20are%20negative%20to%20be%20their%20weight%20multiplied%20by%20%2D1.), Please Update all animals' weights that are negative to be their weight multiplied by -1, not only certain entities you have.
[Hint] It's supposed to be like that:
```sql
UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
```